components/invitation_modal/invitation_modal_confirm_step_row.scss

```diff
@@ -9,6 +9,7 @@
         .email {
             margin: 0 4px 0 8px;
             overflow-wrap: anywhere;
+            word-break: break-word;
```

Comment:
this line is for webkit (chrome and safari)

Reply:
Almost a palindrome :D